recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -8,6 +8,14 @@ XT_GUESTS_BUILD ?= "doma"
 XT_GUESTS_INSTALL ?= "doma"
 
 python __anonymous () {
+    machine = d.getVar('MACHINE', True)
+    bb.warn(machine)
+    not_supported_machines = ["salvator-x-m3", "salvator-xs-h3", "h3ulcb", "m3ulcb", "salvator-x-h3"]
+    if machine in not_supported_machines:
+        warning = 'The machine {} is not supported'.format(machine) 
+        bb.warn(warning)
+        bb.warn("Use branch RELEASE-4.13.0-xt0.1.prod-devel to support.")
```

Comment:
Message needs updating. The branch for which project?

Reply:
added.